charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -367,6 +367,8 @@ spec:
           name: prometheus-bearer-token
           subPath: token
 {{- end }}
+        - mountPath: /var/run/reporting-operator/proxy/user-ca-bundle
```

Comment:
Mounting this volume is contingent on whether or not a cluster-wide Proxy is enabled, and whether the administrator specified an additional CA trust bundle (i.e. `spec.trustedCA`).